package/yast2-firewall.changes

```diff
@@ -1,4 +1,12 @@
 -------------------------------------------------------------------
+Thu Nov  9 09:16:14 UTC 2017 - knut.anderssen@suse.com
+
+- ProposalSettings: fixed method name when vnc is enable by Linuxrc
+  (bsc#1067320)
+- Added tests for installation client.
```

Comment:
I would not mention it in the changes file (tests are not shipped with the package).